channeldb/db.go

```diff
@@ -1168,6 +1176,21 @@ func getLatestDBVersion(versions []version) uint32 {
 	return versions[len(versions)-1].number
 }
 
+// getMinUpgradeVersion returns the minimum version required to upgrade the
+// database.
+func getMinUpgradeVersion(versions []version) uint32 {
```

Comment:
not a blocker, but now that we're removing migrations, this logic should probably be able to handle the case where there are no migrations in a given release. would need a similar change to `getLatestDBVersion` as well